test.properties

```diff
@@ -935,7 +935,7 @@
 
     selenium.chrome.driver.executable=chromedriver${file.suffix.exe}
     selenium.chrome.driver.version[65.0]=2.37
-    selenium.chrome.driver.version[86.0]=86.0.4240.22
+    selenium.chrome.driver.version[86.0]=93.0.4577.63
```

Comment:
@guivilarim, it shouldn't be on this pull. Please always remember to review all content of your commits before send a pull request. :wink: